docs/api/helpers/dialogue/base.md

```diff
@@ -521,11 +521,11 @@ Extend the list of incoming/outgoing messages with 'message', if 'message' belon
 
 True if message successfully added, false otherwise
 
-<a name="aea.helpers.dialogue.base.Dialogue.ensure_counterparty"></a>
```

Comment:
don't bother changing api docs; it's auto generated

Reply:
It was an automatically changed due to PyCharm refactoring. I'll revert back.